src/components/AnonymousChat/child/ChatItemTargetText.tsx

```diff
@@ -1,122 +1,121 @@
     return <View style={styles.mr8}>{avatar}</View>;
   }, []);
 
-  return (
-    <View style={styles.chatContainer}>
-      <View style={styles.wrapper}>
-        {renderAvatar()}
+  const onSwipeToReply = (direction) => {
+    if (direction === 'right') return;
+    if (swipeableRef.current) swipeableRef.current?.close();
+    setReplyPreview({
+      username,
+      time,
+      message,
+      messageId: data?.id,
+      chatType,
+      messageType: 'regular'
+    });
+  };
 
-        <ContextMenu
-          previewBackgroundColor="transparent"
-          style={{flex: 1}}
-          actions={[
-            {title: 'Reply', systemIcon: 'arrow.turn.up.left'},
-            {title: 'Copy Message', systemIcon: 'square.on.square'}
-          ]}
-          onPress={(e) => onContextMenuPressed(e, message)}>
-          <View style={{borderRadius: 8}}>
-            <View style={styles.textContainer}>
-              {!isContinuous && (
-                <View testID="chat-item-user-info" style={styles.chatTitleContainer}>
-                  <Text style={styles.userText}>{username}</Text>
-                  <View style={styles.dot} />
-                  <Text style={styles.timeText}>{time}</Text>
-                </View>
-              )}
+  return (
+    <Swipeable
+      testID="swipeable"
+      ref={swipeableRef}
+      friction={3}
+      overshootFriction={2}
+      onSwipeableOpen={onSwipeToReply}
+      renderLeftActions={replyIcon}>
+      <View style={containerStyle(false, isReplyPrompt)}>
+        <Animated.View style={[styles.wrapper, animatedBubbleStyle]}>
+          {renderAvatar()}
 
-              <Text testID="chat-item-message" style={styles.text}>
-                {message}
-              </Text>
+          <ContextMenu
+            previewBackgroundColor="transparent"
+            style={{flex: 1}}
+            actions={contextMenuActions}
+            onPress={(e) => onContextMenuPressed(e, data, chatType)}>
+            <View style={styles.radius8}>
+              <View style={textContainerStyle(false)}>
+                {isShowUserInfo && (
+                  <View testID="chat-item-user-info" style={styles.chatTitleContainer}>
+                    <Text style={[styles.userText, textStyle(false)]}>{username}</Text>
+                    <View style={dotStyle(false)} />
+                    <Text style={[styles.timeText, textStyle(false)]}>{time}</Text>
+                  </View>
+                )}
+                <ChatReplyView
+                  type={chatType}
+                  messageType={messageType}
+                  replyData={data?.reply_data}
+                />
+                <Text testID="chat-item-message" style={messageStyle(false, messageType)}>
+                  {message}
+                </Text>
+              </View>
             </View>
-          </View>
-        </ContextMenu>
+          </ContextMenu>
+        </Animated.View>
       </View>
-    </View>
+    </Swipeable>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `Swipeable` component is used here to allow users to swipe on a chat item to reply. However, there's no fallback for users who can't perform a swipe gesture. For accessibility reasons, consider adding another way for users to trigger the reply action.
```diff
+   const handleLongPress = () => {
+     setReplyPreview({
+       username,
+       time,
+       message,
+       messageId: data?.id,
+       chatType,
+       messageType: messageType
+     });
+   };
+
    return (
      <Swipeable
        testID="swipeable"
        ref={swipeableRef}
        friction={3}
        overshootFriction={2}
        onSwipeableOpen={onSwipeToReply}
        renderLeftActions={replyIcon}>
+       <View style={containerStyle(false, isReplyPrompt)} onLongPress={handleLongPress}>
          <Animated.View style={[styles.wrapper, animatedBubbleStyle]}>
            {renderAvatar()}
```

Overall, the changes made in this file are significant and improve the chat functionality by introducing swipe to reply feature and displaying replies in a more structured way. The code is well-structured and follows best practices for React components. However, there are some areas where improvements can be made, especially around accessibility and ensuring that the UI updates correctly when certain variables change.


<!-- This is an auto-generated comment by OSS CodeRabbit -->